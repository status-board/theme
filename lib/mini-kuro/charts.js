"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = {
  axis: {
    textColor: '#fff',
    tickColor: '#fff',
    legendColor: '#fff'
  },
  grid: {
    stroke: '#222'
  },
  colors: ['#66c2a5', '#fc8d62', '#8da0cb', '#e78ac3', '#a6d854'],
  tooltip: {
    background: '#000',
    color: '#fff',
    fontSize: '1.4vmin',
    borderRadius: 0,
    boxShadow: 'none',
    border: '1px solid #fff'
  }
};
exports["default"] = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9taW5pLWt1cm8vY2hhcnRzLnRzIl0sIm5hbWVzIjpbImF4aXMiLCJ0ZXh0Q29sb3IiLCJ0aWNrQ29sb3IiLCJsZWdlbmRDb2xvciIsImdyaWQiLCJzdHJva2UiLCJjb2xvcnMiLCJ0b29sdGlwIiwiYmFja2dyb3VuZCIsImNvbG9yIiwiZm9udFNpemUiLCJib3JkZXJSYWRpdXMiLCJib3hTaGFkb3ciLCJib3JkZXIiXSwibWFwcGluZ3MiOiI7Ozs7OztlQUFlO0FBQ1hBLEVBQUFBLElBQUksRUFBRTtBQUNGQyxJQUFBQSxTQUFTLEVBQUUsTUFEVDtBQUVGQyxJQUFBQSxTQUFTLEVBQUUsTUFGVDtBQUdGQyxJQUFBQSxXQUFXLEVBQUU7QUFIWCxHQURLO0FBTVhDLEVBQUFBLElBQUksRUFBRTtBQUNGQyxJQUFBQSxNQUFNLEVBQUU7QUFETixHQU5LO0FBU1hDLEVBQUFBLE1BQU0sRUFBRSxDQUFDLFNBQUQsRUFBWSxTQUFaLEVBQXVCLFNBQXZCLEVBQWtDLFNBQWxDLEVBQTZDLFNBQTdDLENBVEc7QUFVWEMsRUFBQUEsT0FBTyxFQUFFO0FBQ0xDLElBQUFBLFVBQVUsRUFBRSxNQURQO0FBRUxDLElBQUFBLEtBQUssRUFBRSxNQUZGO0FBR0xDLElBQUFBLFFBQVEsRUFBRSxTQUhMO0FBSUxDLElBQUFBLFlBQVksRUFBRSxDQUpUO0FBS0xDLElBQUFBLFNBQVMsRUFBRSxNQUxOO0FBTUxDLElBQUFBLE1BQU0sRUFBRTtBQU5IO0FBVkUsQyIsInNvdXJjZXNDb250ZW50IjpbImV4cG9ydCBkZWZhdWx0IHtcbiAgICBheGlzOiB7XG4gICAgICAgIHRleHRDb2xvcjogJyNmZmYnLFxuICAgICAgICB0aWNrQ29sb3I6ICcjZmZmJyxcbiAgICAgICAgbGVnZW5kQ29sb3I6ICcjZmZmJyxcbiAgICB9LFxuICAgIGdyaWQ6IHtcbiAgICAgICAgc3Ryb2tlOiAnIzIyMicsXG4gICAgfSxcbiAgICBjb2xvcnM6IFsnIzY2YzJhNScsICcjZmM4ZDYyJywgJyM4ZGEwY2InLCAnI2U3OGFjMycsICcjYTZkODU0J10sXG4gICAgdG9vbHRpcDoge1xuICAgICAgICBiYWNrZ3JvdW5kOiAnIzAwMCcsXG4gICAgICAgIGNvbG9yOiAnI2ZmZicsXG4gICAgICAgIGZvbnRTaXplOiAnMS40dm1pbicsXG4gICAgICAgIGJvcmRlclJhZGl1czogMCxcbiAgICAgICAgYm94U2hhZG93OiAnbm9uZScsXG4gICAgICAgIGJvcmRlcjogJzFweCBzb2xpZCAjZmZmJyxcbiAgICB9LFxufTtcbiJdfQ==