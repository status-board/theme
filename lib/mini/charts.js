"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = {
  axis: {
    textColor: '#000',
    tickColor: '#000',
    legendColor: '#000'
  },
  grid: {
    stroke: '#eee'
  },
  colors: ['#b3e2cd', '#fdcdac', '#cbd5e8', '#f4cae4', '#e6f5c9'],
  tooltip: {
    background: '#fff',
    color: '#000',
    fontSize: '1.4vmin',
    borderRadius: 0,
    boxShadow: 'none',
    border: '1px solid #000'
  }
};
exports["default"] = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9taW5pL2NoYXJ0cy50cyJdLCJuYW1lcyI6WyJheGlzIiwidGV4dENvbG9yIiwidGlja0NvbG9yIiwibGVnZW5kQ29sb3IiLCJncmlkIiwic3Ryb2tlIiwiY29sb3JzIiwidG9vbHRpcCIsImJhY2tncm91bmQiLCJjb2xvciIsImZvbnRTaXplIiwiYm9yZGVyUmFkaXVzIiwiYm94U2hhZG93IiwiYm9yZGVyIl0sIm1hcHBpbmdzIjoiOzs7Ozs7ZUFBZTtBQUNYQSxFQUFBQSxJQUFJLEVBQUU7QUFDRkMsSUFBQUEsU0FBUyxFQUFFLE1BRFQ7QUFFRkMsSUFBQUEsU0FBUyxFQUFFLE1BRlQ7QUFHRkMsSUFBQUEsV0FBVyxFQUFFO0FBSFgsR0FESztBQU1YQyxFQUFBQSxJQUFJLEVBQUU7QUFDRkMsSUFBQUEsTUFBTSxFQUFFO0FBRE4sR0FOSztBQVNYQyxFQUFBQSxNQUFNLEVBQUUsQ0FBQyxTQUFELEVBQVksU0FBWixFQUF1QixTQUF2QixFQUFrQyxTQUFsQyxFQUE2QyxTQUE3QyxDQVRHO0FBVVhDLEVBQUFBLE9BQU8sRUFBRTtBQUNMQyxJQUFBQSxVQUFVLEVBQUUsTUFEUDtBQUVMQyxJQUFBQSxLQUFLLEVBQUUsTUFGRjtBQUdMQyxJQUFBQSxRQUFRLEVBQUUsU0FITDtBQUlMQyxJQUFBQSxZQUFZLEVBQUUsQ0FKVDtBQUtMQyxJQUFBQSxTQUFTLEVBQUUsTUFMTjtBQU1MQyxJQUFBQSxNQUFNLEVBQUU7QUFOSDtBQVZFLEMiLCJzb3VyY2VzQ29udGVudCI6WyJleHBvcnQgZGVmYXVsdCB7XG4gICAgYXhpczoge1xuICAgICAgICB0ZXh0Q29sb3I6ICcjMDAwJyxcbiAgICAgICAgdGlja0NvbG9yOiAnIzAwMCcsXG4gICAgICAgIGxlZ2VuZENvbG9yOiAnIzAwMCcsXG4gICAgfSxcbiAgICBncmlkOiB7XG4gICAgICAgIHN0cm9rZTogJyNlZWUnLFxuICAgIH0sXG4gICAgY29sb3JzOiBbJyNiM2UyY2QnLCAnI2ZkY2RhYycsICcjY2JkNWU4JywgJyNmNGNhZTQnLCAnI2U2ZjVjOSddLFxuICAgIHRvb2x0aXA6IHtcbiAgICAgICAgYmFja2dyb3VuZDogJyNmZmYnLFxuICAgICAgICBjb2xvcjogJyMwMDAnLFxuICAgICAgICBmb250U2l6ZTogJzEuNHZtaW4nLFxuICAgICAgICBib3JkZXJSYWRpdXM6IDAsXG4gICAgICAgIGJveFNoYWRvdzogJ25vbmUnLFxuICAgICAgICBib3JkZXI6ICcxcHggc29saWQgIzAwMCcsXG4gICAgfSxcbn07XG4iXX0=