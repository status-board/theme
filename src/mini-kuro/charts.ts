export default {
    axis: {
        textColor: '#fff',
        tickColor: '#fff',
        legendColor: '#fff',
    },
    grid: {
        stroke: '#222',
    },
    colors: ['#66c2a5', '#fc8d62', '#8da0cb', '#e78ac3', '#a6d854'],
    tooltip: {
        background: '#000',
        color: '#fff',
        fontSize: '1.4vmin',
        borderRadius: 0,
        boxShadow: 'none',
        border: '1px solid #fff',
    },
};
