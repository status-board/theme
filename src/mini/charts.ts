export default {
    axis: {
        textColor: '#000',
        tickColor: '#000',
        legendColor: '#000',
    },
    grid: {
        stroke: '#eee',
    },
    colors: ['#b3e2cd', '#fdcdac', '#cbd5e8', '#f4cae4', '#e6f5c9'],
    tooltip: {
        background: '#fff',
        color: '#000',
        fontSize: '1.4vmin',
        borderRadius: 0,
        boxShadow: 'none',
        border: '1px solid #000',
    },
};
