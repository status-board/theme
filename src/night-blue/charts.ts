import colors from './colors';

export default {
    axis: {
        textColor: colors.text,
        tickColor: colors.text,
        legendColor: colors.text,
    },
    grid: {
        stroke: '#25303d',
    },
    colors: ['#FDECA4', '#cbb04a', '#D68649', '#648e9c', '#253445'],
    tooltip: {
        background: '#323f53',
        color: '#f6ecd0',
        fontSize: '1.4vmin',
        borderRadius: '2px',
        boxShadow: '0 1px 3px rgba(0, 0, 0, 0.5)',
    },
};
