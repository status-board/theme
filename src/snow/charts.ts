import colors from './colors';

export default {
    axis: {
        textColor: colors.text,
        tickColor: '#bbb',
        legendColor: colors.text,
    },
    grid: {
        stroke: '#eee',
    },
    colors: ['#82c2de', '#ade0de', '#dad7a3', '#b3deff', '#baab89'],
    tooltip: {
        background: '#fff',
        color: colors.text,
        fontSize: '1.4vmin',
        borderRadius: '2px',
        boxShadow: '0 5px 9px rgba(0, 0, 0, 0.15)',
    },
};
