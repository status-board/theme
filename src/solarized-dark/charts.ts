import colors from './colors';

export default {
    axis: {
        textColor: colors.text,
        tickColor: colors.text,
        legendColor: colors.text,
    },
    grid: {
        stroke: '#073642',
    },
    colors: ['#00b2b0', '#24dead', '#d9d356', '#f1c4c2', '#dfbbe8'],
    tooltip: {
        background: '#073c49',
        color: '#bed0d2',
        fontSize: '1.4vmin',
        borderRadius: 0,
        boxShadow: '0 1px 3px rgba(0, 0, 0, 0.5)',
    },
};
