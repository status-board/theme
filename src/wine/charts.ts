import colors from './colors';

export default {
    axis: {
        textColor: colors.text,
        tickColor: colors.text,
        legendColor: colors.text,
    },
    grid: {
        stroke: '#321515',
    },
    colors: ['#a52818', '#e46a55', '#bd532e', '#9b2c13', '#E43D24'],
    tooltip: {
        background: '#501919',
        color: 'hsl(10, 60%, 90%)',
        fontSize: '1.4vmin',
        borderRadius: 0,
        boxShadow: '0 2px 5px rgba(0, 0, 0, 0.6)',
    },
};
