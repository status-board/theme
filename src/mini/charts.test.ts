import charts from './charts';

test('Charts should match snapshot', (): void => {
    expect(charts).toMatchSnapshot();
});
