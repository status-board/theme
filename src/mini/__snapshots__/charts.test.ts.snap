// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charts should match snapshot 1`] = `
Object {
  "axis": Object {
    "legendColor": "#000",
    "textColor": "#000",
    "tickColor": "#000",
  },
  "colors": Array [
    "#b3e2cd",
    "#fdcdac",
    "#cbd5e8",
    "#f4cae4",
    "#e6f5c9",
  ],
  "grid": Object {
    "stroke": "#eee",
  },
  "tooltip": Object {
    "background": "#fff",
    "border": "1px solid #000",
    "borderRadius": 0,
    "boxShadow": "none",
    "color": "#000",
    "fontSize": "1.4vmin",
  },
}
`;
